gapic/core/artman_logging_types.yaml

```diff
@@ -0,0 +1,16 @@
+common:
+  api_name: service-logging-types
+  api_version: v1
+  organization_name: google
+  proto_gen_pkg_deps:
+    - google-common-protos
+  import_proto_path:
+    - ${REPOROOT}/googleapis
+  src_proto_path:
+    # Remaining AuditData types should be added here. See
```

Comment:
You could also spec out what the other lines will be and just comment them, so it's like a checklist for what remains to remove this TODO comment.